doc/examples.rst

```diff
@@ -33,6 +33,8 @@ other sources.
    steering-gainsched
    steering-optimal
    kincar-flatsys
+   mrac_siso_direct_mit_rule_statespace
+   mrac_siso_direct_lya_rule_statespace
```

Comment:
Perhaps use shorter filenames?  It seems like `mrac_siso_mit` and `mrac_siso_lyapunov` would fit the bill.